views/index.html

```diff
@@ -4,24 +4,109 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Blank</title>
+    <title>Never Bored of Board Games</title>
 
     <!-- STYLESHEETS -->
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
     <link rel="stylesheet" href="/styles/styles.css">
 
+    <!-- fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Baloo+Tammudu" rel="stylesheet">
+
     <!-- VENDOR SCRIPTS -->
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
 
     <!-- APPLICATION SCRIPTS -->
     <script src="/scripts/app.js"></script>
   </head>
   <body>
+
+    <div class ="top-banner">
+      <h1 class="main-title">Board Games</h1>
+      <h4><a href="/api">Read My API Documentation</a></h4>
+    </div>
+
+    <div class="background">
+
+
+
+
     <div class="container">
       <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-          <h1>Under Construction</h1>
-          <h4><a href="/api">Read My API Documentation</a></h4>
+
+
+
+        <!-- Button to add new board games -->
+        <div class="col-md-4 col-md-offset-4" align="center">
+          <button class="btn btn-success new-game">Add New Board Game</button>
+        </div>
+
+        <!-- Start of new game form -->
+        <div class="col-md-4 col-md-offset-4 hidden-form">
+
+          <form class="new-game-form">
+
+            <div class="form-group row title-input">
+              <label for="example-text-input" class="col-2 col-form-label">Title</label>
+              <div class="col-10">
+                <input class="form-control" type="text" name="title">
+              </div>
+            </div>
+
+            <div class="form-group">
+              <label>Description</label>
+              <textarea name="description" class="form-control" rows="3"></textarea>
+            </div>
+
+            <div class="form-group row">
+              <label for="example-text-input" class="col-2 col-form-label">Playtime</label>
+              <div class="col-10">
+                <input class="form-control" type="text" name="playtime">
+              </div>
+            </div>
+
+            <div class="form-group row">
+              <label for="example-text-input" class="col-2 col-form-label">Players</label>
+              <div class="col-10">
+                <input class="form-control" type="text" name="players">
+              </div>
+            </div>
+
+            <div class="form-group row">
+              <label for="example-text-input" class="col-2 col-form-label">Image</label>
```

Comment:
label `for` is supposed to be for the `id` of the input the label points at--you need to add IDs to the inputs and make the `for` equal to that ID.